### PR TITLE
fix (Simulator) use stdout to get results from ios-sim

### PR DESCRIPTION
**Pull request by pentateu**
This PQ fixes the 200% CPU use of ios-sim

PRE-REQ:  update our   ios-sim repo to the latest version

### CODE REVIEW

File: src/steroids/Simulator.coffee [hunk 1]
```diff
@@ -92,7 +90,7 @@ class Simulator
             # Split into device type and optional, '@'-separated suffix specifying the iOS version (SDK version; e.g., '5.1').
             [device, iOSVersion] = opts.device.split('@')
 
-          steroidsCli.log "Starting #{device} Simulator"
+          steroidsCli.log "Starting #{device} Simulator X3"
```
Comment (harsaharsa) [inline src/steroids/Simulator.coffee]: Why this change?

Comment (pentateu) [inline src/steroids/Simulator.coffee]: have no idea.. will remove
